packages/quantic/cypress/integration/facets/category-facet/category-facet-selectors.ts

```diff
@@ -1,36 +1,30 @@
 import {ComponentSelector, CypressSelector} from '../../common-selectors';
 import {
+  BaseFacetSelector,
```

Comment:
This file contains a lot of selectors that appear to be common to many facets (clearFilter, placeholder, facetCount, showLess, showMore, expandButton, collapseButton, etc). Is it possible to reduce this duplication?

Reply:
I don't get your comment

Reply:
This file doesn't seem to be utilizing some of the common selectors we defined in `facet-common-selectors.ts`.

Reply:
Maybe if we had something like this in the common-selectors file.

```
export function commonSelectors(get: () => CypressSelector) {
  return {
    placeholder: () => get().find('.placeholder__card-container'),
  }
}
```

and use it like this

```
export const CategoryFacetSelectors: AllFacetSelectors = {
  get: () => cy.get(categoryFacetComponent),
  ...commonSelectors(CategoryFacetSelectors.get),
}
```